src/sql/base/browser/ui/table/asyncDataView.ts

```diff
@@ -179,6 +183,10 @@ export class VirtualizedCollection<T extends Slick.SlickData> implements IObserv
 		return this.placeHolderGenerator(index);
 	}
 
+	public publicResetWindowsAroundIndex(index: number): void {
```

Comment:
if you are going to make this public, I would just change the original function from private to public